Modules/Forum/classes/class.ilForumCronNotification.php

```diff
@@ -73,6 +92,35 @@ public function __construct(\ilDBInterface $database = null, \ilForumNotificatio
 			$notificationCache = new \ilForumNotificationCache();
 		}
 		$this->notificationCache = $notificationCache;
+
+		if ($ilAccess === null) {
+			global $DIC;
```

Comment:
@legionth Using the `$DIC` in the constructor of `ilCronJob` derivates could be an issue when reloading the `ilCtrl` structure in the ILIAS setup.